addins/Cake.APT.Module.yml

```diff
@@ -0,0 +1,16 @@
+Name: Cake.APT.Module
```

Comment:
@Jericho I thought we weren’t including modules?

Reply:
I know, I forgot.

Reply:
😄